docs/source/guides/db_integration_with_featuretools_sql.ipynb

```diff
@@ -0,0 +1,125 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# What is `featuretools_sql`? \n",
```

Comment:
```suggestion
    "# Connecting to SQL \n",
```

Reply:
should we make this match the title of the file?

Reply:
Yes

Reply:
sounds good, going with `database integration` then

Reply:
Maybe do **SQL Database Integration**? What do you think?

Reply:
that works too